source/isl/domain.c

```diff
@@ -1476,6 +1476,39 @@ CloogDomain *cloog_domain_cube(CloogState *state,
 	return cloog_domain_from_isl_set(cube);
 }
 
+/**
+ * cloog_domain_from_vec
+ * Construct and return a dim-dimensional cube, with values ranging
+ * between lower_bounds and upper_bounds in each dimension.
+ */
+CloogDomain *cloog_domain_from_vec(CloogState *state,
```

Comment:
- in the general comment: "dim-dimensional": please tell where it is (x_bounds->size),
- mixed space-tabulation indentation... I would suggest to conform to Google C++ style,
- a better naming would be cloog_domain_from_bounds.